src/dotnet/commands/dotnet-cache/LocalizableStrings.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Microsoft.DotNet.Tools.Cache
+{
+    internal class LocalizableStrings
+    {
+        public const string AppFullName = ".NET Cache";
+
+        public const string AppDescription = "Cahes Assemblies specified for the .NET Platform, By Default these will be optimized for the target runtime and framework";
+
+        public const string ProjectEntries = "ProjectEntries";
+
+        public const string ProjectEntryDescription = "The MSBuild project file used to specify the list of packages to be cached.";
```

Comment:
Is it important to say is a MSBuild project file? Isn't just project file enough?

Wouldn't this be better as "The project file that contains the list of packages to be cached."?